src/main/java/org/petmarket/cart/entity/Cart.java

```diff
@@ -0,0 +1,78 @@
+package org.petmarket.cart.entity;
+
+import jakarta.persistence.*;
+import lombok.*;
+import org.petmarket.users.entity.User;
+
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+@Entity
+@Table(name = "cart")
+@Getter
+@Setter
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class Cart {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id", nullable = false)
+    private Long id;
+
+    @ManyToOne
```

Comment:
Тут якщо я правильно розумію має бути @OneToOne 
```suggestion
    @OneToOne
```

Reply:
виправив